Realm/RLMObservation.mm

```diff
@@ -571,15 +571,15 @@ static KeyPath keyPathFromString(RLMRealm *realm,
         TableKey tk = info->objectSchema->table_key;
         ColKey ck;
         if (property.type == RLMPropertyTypeObject) {
-            ck = info->tableColumn(property.columnName);
+            ck = info->tableColumn(property.name);
```

Comment:
I think this change is correct, but it suggests we're missing some KVO tests that should be added.

Reply:
Added Test for this in objective-C, and there were some already in Swift in the new ColumnNameTest suite